src/utils/utils.ts

```diff
@@ -108,7 +108,16 @@ export const generateItemFromFilename = async (options: {
     const { size } = stats;
 
     // upload file
-    const uploadFilePath = await uploadFile({ mimetype, filepath });
+    let uploadFilePath = null;
+    try {
+      uploadFilePath = await uploadFile({ mimetype, filepath });
+    } catch (e) {
```

Comment:
This will also catch any other kind of error and silence it, is this behaviour expected?